jobs/snort/templates/rules/local.rules.erb

```diff
@@ -1 +1,3 @@
-<%= p('snort.rules', '') %>
+<% p("snort.rules", []).each do |rule| %>
+  <%= rule %>
```

Comment:
This was just absent-minded, reflexive indentation on my part that was incorrect. Thanks for catching it --